config/oidc.php

```diff
@@ -3,10 +3,85 @@
 declare(strict_types=1);
 
 return [
+    /**
+     * The issuer URL of the OpenID Connect provider.
+     */
     'issuer' => env('OIDC_ISSUER', ''),
+
+    /**
+     * The client ID of the OpenID Connect provider.
```

Comment:
I think the `client_id` refers to the application unique id and not the provider

Reply:
Indeed. Good one, will change it at the original place in the package.